src/dune_rules/super_context.ml

```diff
@@ -2,13 +2,16 @@ open! Dune_engine
 open! Stdune
 open Import
 
-let default_context_flags (ctx : Context.t) =
-  (* TODO DUNE3 To ensure full backward compatibility, ocaml_cflags are still
```

Comment:
I think there should be a TODO DUNE3 comment somewhere about making the opt-in behavior added in the PR a dune 3 default.